src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/test_appconfig_commands.py

```diff
@@ -951,6 +951,7 @@ def test_azconfig_import_export(self, resource_group, location):
         import_features_alt_syntax_file_path = os.path.join(TEST_DIR, 'import_features_alt_syntax.json')
         import_features_random_conditions_file_path = os.path.join(TEST_DIR, 'import_features_random_conditions.json')
         import_features_invalid_requirement_type_file_path = os.path.join(TEST_DIR, 'import_features_invalid_requirement_type.json')
+        os.environ['AZURE_APPCONFIG_FM_COMPATIBLE'] = 'True'
```

Comment:
If the test suite is running tests in parallel, environment variables may get mixed up between tests. When setting up tests that depend on environment variables, we need to make sure that those tests never run in parallel. Is that something we need to do for CLI tests?

Reply:
This is a good point. I have taken a look at how the tests run, it seems the tests run sequentially. The tests are grouped into classes and within the class we have functions with the tests. When a test within the function fails the class is marked as failed. It looks like we only queue the next function in a given class after the current running test in the class has passed/has been skipped. 

Discussed with @albertofori as well. We think we can at least group tests that depend on the same value of the environment variable together, please let me know what you think.